tests/workflows/helloWorld.wdl

```diff
@@ -77,14 +77,14 @@ task HelloWorldTask {
 
     # ------------------------------------------------
     # Runtime settings:
-     runtime {
-         docker: docker
-         memory: machine_mem + " MB"
-         disks: "local-disk " + select_first([disk_space_gb, default_disk_space_gb]) + if use_ssd then " SSD" else " HDD"
-         bootDiskSizeGb: select_first([boot_disk_size_gb, default_boot_disk_size_gb])
-         preemptible: 0 
-         cpu: select_first([cpu, 1])
-     }
+#     runtime {
```

Comment:
The installed Cromwell shouldn't be using these parameters since it hasn't been configured for the cloud, but just in case commenting them out to avoid unnecessary failures